include/dgl/graph_interface.h

```diff
@@ -333,6 +333,10 @@ class GraphInterface {
    * \return a vector of IdArrays.
    */
   virtual std::vector<IdArray> GetAdj(bool transpose, const std::string &fmt) const = 0;
+
+  virtual std::vector<IdArray> GetNodeFlowSlice(bool transpose, const std::string &fmt,
+                                                size_t layer0_size, size_t layer1_start,
+                                                size_t layer1_end, bool remap) const = 0;
```

Comment:
Let's try not add more APIs to `GraphInterface` header files, esp. APIs for nodeflow.